uint8 COLOR_RED = 0
uint8 COLOR_BLUE = 1
uint8 COLOR_YELLOW = 2
uint8 color

# The size of the brick is the number of holes (4,6 or 8)
uint8 size
uint8 count